### PR TITLE
Update Tests to NServiceBus 7.1 and .NET Core 2.1

### PR DESCRIPTION
**Pull request by timbussmann**

### CODE REVIEW

File: src/NServiceBus.Transport.RabbitMQ.AcceptanceTests/NServiceBus.Transport.RabbitMQ.AcceptanceTests.csproj [hunk 1]
```diff
@@ -11,9 +11,9 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0" />
-    <PackageReference Include="NServiceBus.AcceptanceTests.Sources" Version="7.0.0" />
-    <PackageReference Include="NUnit" Version="3.8.1" />
-    <PackageReference Include="NUnit3TestAdapter" Version="3.8.0-alpha1" />
+    <PackageReference Include="NServiceBus.AcceptanceTests.Sources" Version="7.1.0" />
+    <PackageReference Include="NUnit" Version="3.10.1" />
+    <PackageReference Include="NUnit3TestAdapter" Version="3.10.0" />
```
Comment (timbussmann) [inline src/NServiceBus.Transport.RabbitMQ.AcceptanceTests/NServiceBus.Transport.RabbitMQ.AcceptanceTests.csproj]: @bording shouldn't be an issue as long as we have no usage of mono.cecil I think?

**Comment by adamralph** [comment]
@bording I assume you are acting as reviewer on this one?

File: src/NServiceBus.Transport.RabbitMQ.AcceptanceTests/NServiceBus.Transport.RabbitMQ.AcceptanceTests.csproj (hunk 1 above)
Comment (bording) [reply src/NServiceBus.Transport.RabbitMQ.AcceptanceTests/NServiceBus.Transport.RabbitMQ.AcceptanceTests.csproj]: As we discussed, this one is fine.

File: src/NServiceBus.Transport.RabbitMQ.Tests/NServiceBus.Transport.RabbitMQ.Tests.csproj [hunk 2]
```diff
@@ -15,11 +15,11 @@
     <PackageReference Include="ApprovalTests" Version="3.0.13" NoWarn="NU1701" />
     <PackageReference Include="ApprovalUtilities" Version="3.0.13" NoWarn="NU1701" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0" />
-    <PackageReference Include="NServiceBus" Version="7.0.0" />
-    <PackageReference Include="NUnit" Version="3.8.1" />
-    <PackageReference Include="NUnit3TestAdapter" Version="3.8.0-alpha1" />
+    <PackageReference Include="NServiceBus" Version="7.1.0" />
+    <PackageReference Include="NUnit" Version="3.10.1" />
+    <PackageReference Include="NUnit3TestAdapter" Version="3.10.0" />
```
Comment (bording) [inline src/NServiceBus.Transport.RabbitMQ.Tests/NServiceBus.Transport.RabbitMQ.Tests.csproj]: This one however, can't be updated.

Comment (bording) [inline src/NServiceBus.Transport.RabbitMQ.Tests/NServiceBus.Transport.RabbitMQ.Tests.csproj]: This one shouldn't be updated.

File: src/NServiceBus.Transport.RabbitMQ.Tests/NServiceBus.Transport.RabbitMQ.Tests.csproj [hunk 3]
```diff
@@ -15,11 +15,11 @@
...
     <PackageReference Include="RabbitMQ.Client" Version="5.0.1" />
-    <PackageReference Include="PublicApiGenerator" Version="6.5.0" />
+    <PackageReference Include="PublicApiGenerator" Version="8.0.0" />
```
Comment (bording) [inline src/NServiceBus.Transport.RabbitMQ.Tests/NServiceBus.Transport.RabbitMQ.Tests.csproj]: While there's probably no harm in updating this, we don't need this to be part of the PR.

**Comment by timbussmann** [comment]
updated as discussed

**Pull request merged**